tools/harness-thci/OpenThread.py

```diff
@@ -1805,15 +1805,16 @@ def configBorderRouter(
         if TESTHARNESS_VERSION == TESTHARNESS_1_2 and P_dp == 0 and not self.__isOpenThreadRunning():
             self.__useDefaultDomainPrefix = False
 
-        if TESTHARNESS_VERSION == TESTHARNESS_1_2:
+        if TESTHARNESS_VERSION == TESTHARNESS_1_2 and self.IsBorderRouter:
```

Comment:
```suggestion
        if TESTHARNESS_VERSION == TESTHARNESS_1_2 and self.IsBackboneRouter:
```

Every device using OpenThread_BR.py THCI will be `IsBorderRouter`, but a BR can also be DUT for 1.1 certification. So, we need to use 1.1 prefix rules for such BR device.

Reply:
Tested and working.  Thanks, Simon!

Reply:
`configBorderRouter` is defined in class `OpenThreadTHCI`. Isn't OpenThreadTHCI used by both OpenThread.py and OpenThread_BR.py ?